CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+# Default
+*                          @google/certificate-transparency
```

Comment:
Shouldn't that be this repo, not the C++ one?

Reply:
I think that's a [team](https://github.com/orgs/google/teams/certificate-transparency/members) not a repo (confusingly)

Reply:
Ah, it confused me. Never mind then.